ci-operator/config/openshift-kni/cnf-features-deploy/openshift-kni-cnf-features-deploy-master.yaml

```diff
@@ -3,14 +3,18 @@ base_images:
     name: "4.9"
     namespace: ocp
     tag: base
+  ocp_builder_rhel-8-golang-1.15-openshift-4.7:
```

Comment:
4.7?
I was able to find one without ocp affinity:
`registry.svc.ci.openshift.org/openshift/release:golang-1.13`
maybe there's one for 1.15, 1.16 too

Reply:
I am using the same config as all the other operators for example:
sriov - https://github.com/openshift/release/blob/master/ci-operator/config/openshift/sriov-network-operator/openshift-sriov-network-operator-master.yaml
ptp - https://github.com/openshift/release/blob/master/ci-operator/config/openshift/ptp-operator/openshift-ptp-operator-master.yaml

we can open a follow up PR to bump the version in the repo and then add new images here as for example the machine-config-operator

https://github.com/openshift/release/blob/master/ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-master.yaml#L49

Reply:
IMHO we should use the right version rhel-8-golang-1.16-openshift-4.9 from the beginning.

Reply:
And the same is true for SR-IOV and PTP operators.

Reply:
If we do a re-upload, I agree we should change.

Reply:
we also change the images on cnf-feature-deploy so we must update :)

Reply:
Thanks!